frontend/src/modules/initiative/view.jsx

```diff
@@ -859,14 +860,17 @@ const getSchema = (
 const AddInitiative = ({ match: { params }, ...props }) => {
   const minHeightContainer = innerHeight * 0.8;
   const minHeightCard = innerHeight * 0.75;
+  const globalState = UIStore.useState();
```

Comment:
same here https://lostpebble.github.io/pullstate/docs/use-store-state-hook#use-a-sub-state-of-a-store